src/Config/container.ts

```diff
@@ -59,12 +59,17 @@ import {
   DefaultRuntimeDetector
 } from '../Repeater';
 import { container, Lifecycle } from 'tsyringe';
+import path from 'path';
 
 container
   .register('tsyringe', {
     useValue: container
   })
-  .register<CliInfo>(CliInfo, { useValue: new CliInfo(process.cwd()) })
+  .register<CliInfo>(CliInfo, {
+    useValue: new CliInfo(
+      (require.main && path.dirname(require.main.filename)) ?? __dirname
```

Comment:
Why don't use `__dirname` instead? `dirname(require.main.filename)` and `__dirname` are equivalent for the main entry

Reply:
You are right, In our case it makes sense to use only `__dirname`